### PR TITLE
Add Alibaba Cloud Nacos binding Doc

### LINKED ISSUE #1400: [CONTENT]Alibaba Cloud Nacos Binding Component
**What content needs to be created or modified?**
Alibaba Cloud Nacos Binding Component

**Additional context**
dapr/components-contrib#810

### PR DESCRIPTION
**Pull request by yanjianlong**
Thank you for helping make the Dapr documentation better!

**Please follow this checklist before submitting:**

- [x] [Read the contribution guide](https://docs.dapr.io/contributing/contributing-docs/)
- [x] Commands include options for Linux, MacOS, and Windows within codetabs
- [x] New file and folder names are globally unique
- [x] Page references use shortcodes instead of markdown or URL links
- [x] Images use HTML style and have alternative text
- [x] Places where multiple code/command options are given have codetabs

In addition, please fill out the following to help reviewers understand this pull request:

## Description

Added docs for Alibaba Cloud Nacos binding

## Issue reference
#1400

### CODE REVIEW

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md [hunk 1]
```diff
@@ -0,0 +1,115 @@
+---
+type: docs
+title: "Alibaba Cloud Nacos binding spec"
+linkTitle: "Alibaba Cloud Nacos"
+description: "Detailed documentation on the Alibaba Cloud Nacos binding component"
+aliases:
+- "/operations/components/setup-bindings/supported-bindings/alicloud-nacos/"
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md]:
```suggestion
```

You can delete this as it's the first time this page is getting created and there wasn't a prior location.

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md [hunk 2]
```diff
@@ -0,0 +1,115 @@
...
+
+This component supports both **input and output** binding interfaces.
+
+This component supports **output binding** with the following operations:
+- `create`
+- `get`
+
+## Create an AliCloud Nacos
+Follow the instructions [here](https://www.alibabacloud.com/help/doc-detail/59963.htm) on setting up AliCloud nacos.
+Follow the instructions [here](https://help.aliyun.com/document_detail/85466.html) on setting up AliCloud nacos.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md]:
```suggestion
Follow [these instructions](https://www.alibabacloud.com/help/doc-detail/59963.htm) to setup AliCloud Nacos.
   - A Chinese (simplified) version can be found [here](https://help.aliyun.com/document_detail/85466.html)
```

Added suggestion for grammer. But curious why these links go to ACM. Shouldn't the Nacos link be https://nacos.io/en-us/docs/quick-start.html?

File: daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md [hunk 3]
```diff
@@ -0,0 +1,115 @@
...
+
+## Specifying a partition key
+
+When invoking the nacos binding, its possible to provide an optional partition key by using the `metadata` section in the request body.
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/alicloud-nacos.md]:
```suggestion
When invoking the Nacos binding, its possible to provide an optional partition key by using the `metadata` section in the request body.
```

**Review by AaronCrawfis — CHANGES_REQUESTED** [review]

File: daprdocs/content/en/reference/components-reference/supported-bindings/_index.md [hunk 4]
```diff
@@ -46,9 +46,10 @@ Table captions:
 
 ### Alibaba Cloud
 
-| Name | Input<br>Binding | Output<br>Binding | Status |
-|------|:----------------:|:-----------------:|--------|
-| [Alibaba Cloud OSS]({{< ref alicloudoss.md >}})           |    | ✅ | Alpha |
+| Name | Input<br>Binding | Output<br>Binding | Status | Component version | Since |
+|------|:----------------:|:-----------------:|--------| ------ |----------|
+| [Alibaba Cloud OSS]({{< ref alicloudoss.md >}})           |    | ✅ | Alpha |  v1 | 1.0 |
+| [Alibaba Cloud Nacos]({{< ref alicloud-nacos.md >}})           |  ✅ | ✅ | Alpha |  v1 | 1.0 |
```
Comment (AaronCrawfis) [inline daprdocs/content/en/reference/components-reference/supported-bindings/_index.md]:
```suggestion
| [Alibaba Cloud Nacos]({{< ref alicloud-nacos.md >}})           |  ✅ | ✅ | Alpha |  v1 | 1.2 |
```

**Comment by orizohar** [comment]
Waiting for code PR https://github.com/dapr/components-contrib/pull/814

**Review by AaronCrawfis — APPROVED** [review]

**Pull request merged**